### PR TITLE
VirtualKeyBoard: Switch to schedule-less visual feedback for flash_keyboard

### PR DESCRIPTION
**Pull request by NiLuJe**
Re: #8089

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8143)

### CODE REVIEW

**Comment by NiLuJe** [comment]
Fixes the lingering vertical ellipsis thingy mentioned, and hopefully doesn't break anything in the process ;).

**Comment by NiLuJe** [comment]
(As for the actual behavior, widget/button is where this stuff is documented)

**Comment by NiLuJe** [comment]
As for the flashing merge issue mentioned (that was on a H2O):

```
[pid  3119] 15:32:37.738224 [2ac7a4ec] ioctl(4</dev/fb0>, MXCFB_SEND_UPDATE_V1_NTX, {update_region={top=1325, left=702, width=103, height=110}, waveform_mode=WAVEFORM_MODE_AUTO, update_mode=UPDATE_MODE_PARTIAL, update_marker=38, temp=TEMP_USE_AMBIENT, flags=0, alt_buffer_data={virt_addr=NULL, phys_addr=NULL, width=0, height=0, alt_update_region={top=0, left=0, width=0, height=0}}}) = 0 <0.009706>
[pid  3119] 15:32:37.806561 [2ac7a4ec] ioctl(4</dev/fb0>, MXCFB_SEND_UPDATE_V1_NTX, {update_region={top=864, left=0, width=1080, height=576}, waveform_mode=WAVEFORM_MODE_AUTO, update_mode=UPDATE_MODE_FULL, update_marker=39, temp=TEMP_USE_AMBIENT, flags=0, alt_buffer_data={virt_addr=NULL, phys_addr=NULL, width=0, height=0, alt_update_region={top=0, left=0, width=0, height=0}}}) = 0 <0.000514>
[pid  3119] 15:32:37.808020 [2ac7a4ec] ioctl(4</dev/fb0>, MXCFB_WAIT_FOR_UPDATE_COMPLETE_V1, 39) = 878 <1.213802>
[pid  3119] 15:32:39.033409 [2ac7a4ec] ioctl(4</dev/fb0>, MXCFB_SEND_UPDATE_V1_NTX, {update_region={top=416, left=151, width=100, height=47}, waveform_mode=WAVEFORM_MODE_AUTO, update_mode=UPDATE_MODE_PARTIAL, update_marker=40, temp=TEMP_USE_AMBIENT, flags=0, alt_buffer_data={virt_addr=NULL, phys_addr=NULL, width=0, height=0, alt_update_region={top=0, left=0, width=0, height=0}}}) = 0 <0.011491>
```

The third refresh gets merged into a *copy* of the second one (instead of the second one proper, or being left alone), which means we get two flashes :/.

**Review by Frenzie — APPROVED** [review]
Alright.

File: frontend/ui/widget/virtualkeyboard.lua [hunk 1]
```diff
@@ -343,12 +346,15 @@ function VirtualKey:onTapSelect(skip_flash)
     -- just in case it's not flipped to false on hold release where it's supposed to
     self.keyboard.ignore_first_hold_release = false
     if self.flash_keyboard and not skip_flash and not self.skiptap then
-        self[1].inner_bordersize = self.focused_bordersize
```
Comment (poire-z) [inline frontend/ui/widget/virtualkeyboard.lua:346]: No more inner border stuff anywhere ? What's flashing then ?

You're also disabling that here in onTapSelect and onSwipeSelect below ?

File: frontend/ui/widget/virtualkeyboard.lua [hunk 2]
```diff
@@ -358,17 +364,22 @@ function VirtualKey:onTapSelect(skip_flash)
 end
 
 function VirtualKey:onHoldSelect()
+    print("OnHoldSelect on", self)
```
Comment (poire-z) [inline frontend/ui/widget/virtualkeyboard.lua:367]: Forgot to remove it.

**Comment by poire-z** [comment]
May be merge this only after #8142, to avoid conflicts and @hius07 having to rebase/fix via GH UI.

File: frontend/ui/widget/virtualkeyboard.lua (hunk 1 above)
Comment (NiLuJe) [reply frontend/ui/widget/virtualkeyboard.lua:346]: It's handled in the `invert` method ;).

**Review by poire-z — APPROVED** [review]

**Comment by hius07** [comment]
Yeah, very nice everywhere.

**Pull request merged**